src/test/resources/features/create_employee.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Create Employee
```

Comment:
Put all the feature scenarios in one file

Reply:
done as part of 8aea984